templates/interface/filter-panels.interface.hbs

```diff
@@ -121,13 +122,14 @@
           <label for="value-type-field-{{id}}" class="select-proxy-display">
             <select name="value-type-field-{{id}}" id="value-type-field-to-{{id}}" data-field="valueType" class="hidden-select form-control">
               <option value="enter-value">Enter a value</option>
+              <option value="user-profile-data">User profile data</option>
             </select>
             <span class="icon fa fa-chevron-down"></span>
           </label>
         </div>
       </div>
 
-      <div class="form-group clearfix">
+      <div class="form-group clearfix" id="filter-value-between-{{id}}">
```

Comment:
There are two elements on the page with the same ID. We can't have that.

Reply:
@tonytlwu fixed